doc/phpactor.txt

```diff
@@ -191,20 +191,22 @@ Phpactor does not assume any mappings automatically, the following mappings
 are available for you to copy:
 >
 
-  nmap <Leader>u :PhpactorImportClass<CR>
-  nmap <Leader>ua :PhpactorImportAllClasses<CR>
```

Comment:
This sohuld have been `:PhpactorImportMissingClasses`

Reply:
Added `:PhpactorImportMissingClasses` mapping